### PR TITLE
Use logging filter, remove otel.

### PR DESCRIPTION
**Pull request by yuroitaki**
This addresses https://github.com/tlsnotary/tlsn/issues/412

- ~Switched from using config file to indicate log verbosity level to using RUST_LOG env var which is much more flexible~
- Removed opentelemetry setup code which is flooding the logs with `SpanData` struct because it was temporarily setup to export to stdout — we can remove it for now since we're not using any opentelemetry tool yet
- Fixed the config file logging setting to use logging filter directive instead of just verbosity level

### CODE REVIEW

File: notary-server/README.md
Comment (heeckhau) [inline notary-server/README.md:87]: Maybe we should call this section `Logging`?

**Review by heeckhau — CHANGES_REQUESTED** [review]

File: components/tls/tls-client/src/backend/standard.rs
Comment (yuroitaki) [reply components/tls/tls-client/src/backend/standard.rs:1]: yep ran cargo fmt and affected them, but have excluded them

**Comment by yuroitaki** [comment]
By the way @sinui0 @heeckhau please don't merge this first even after all approvals have been obtained when I am away — since there's a format change of the config file, I'd like to change that in our AWS server first before CD kicks in which ll fail otherwise (though that's actually okay since we can manually start the server after, just my preference to correct things before CD fails)

File: notary-server/README.md
Comment (heeckhau) [inline notary-server/README.md:87]:
```suggestion
The default logging strategy of this server is set to `DEBUG` verbosity level for the crates that are useful for most debugging scenarios, i.e. using the following filtering logic:
```

**Review by heeckhau — APPROVED** [review]

**Pull request merged**
